client/src/components/SpinnerWrapper/SpinnerWrapper.js

```diff
@@ -3,7 +3,8 @@ import React from "react";
 
 import { Spinner, spinner_configs } from "src/core/Spinner.js";
 
-import { LeafSpinner } from "./LeafSpinner.js";
+// eslint-disable-next-line no-restricted-imports
+import { LeafSpinner } from "../LeafSpinner.js";
```

Comment:
I think it's fair to rename folder name `SpinnerWrapper` to `Spinner` and include LeafSpinner all together !